docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,442 @@ card(
+    const anchorRef = React.useRef(null);
+    const handleSelect = ({ item }) => {
+      setSelected(item);
+    };
+
+    return (
+      <Flex gap={2}>
+      <SelectList
+        id="selectlistexample"
+        onChange={() => {}}
+        options={[
+          {label:'Impressions', value: 'v1'},
+          {label:'Pin clicks', value: 'v2'},
+          {label:'Outbound clicks', value: 'v3'},
+          {label:'Saves', value: 'v4'},
+          {label:'Engagement rate', value: 'v5'},
+        ]}
+        size='lg'
+      />
+      <Box display="flex" justifyContent="center">
+
+        <Button
+          accessibilityControls="header-dropdown-example"
+          accessibilityHaspopup
+          accessibilityExpanded={ open }
+          iconEnd="arrow-down"
+          text="Menu"
+          inline
+          selected={open}
+          icon="add"
+          size="lg"
+          onClick={ () => setOpen((prevVal) => !prevVal) }
+          ref={anchorRef}
+        />
+        {open && (
+          <Dropdown id="subtext-dropdown-example" anchor={anchorRef.current} onDismiss={() => {setOpen(false)}}>
+            <Dropdown.Item
+              handleSelect={handleSelect}
+              selected={selected}
+              option={{ value: "item 2", label: "Engagement" }}
+            />
+            <Dropdown.Item
+              handleSelect={handleSelect}
+              selected={selected}
+              option={{ value: "item 2", label: "Pin clicks" }}
+            />
+            <Dropdown.Item
+              handleSelect={handleSelect}
+              selected={selected}
+              option={{ value: "item 2", label: "Outbound clicks" }}
+            />
+            <Dropdown.Item
+              handleSelect={handleSelect}
+              selected={selected}
+              option={{ value: "item 2", label: "Saves" }}
+            />
+            <Dropdown.Item
+              handleSelect={handleSelect}
+              selected={selected}
+              option={{ value: "item 2", label: "Engagement rate" }}
+            />
+          </Dropdown>
+        )}
+      </Box>
+      </Flex>
+    );
+  }
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="Labels"
+      description={`
+      SelectList comes with Label built-in: just use the \`label\` prop. Be sure to provide a unique \`id\` so the Label is associated with the correct SelectList.`}
     />
-  );
-}
-    `}
-  />,
+  </MainSection>,
 );
 
 card(
-  <Example
-    id="helperText"
-    name="Example: Helper Text"
-    description={`Whenever you want to provide more information about a form field, you should use \`helperText\`.`}
-    defaultCode={`
+  <MainSection name="Variants">
+    <MainSection.Subsection title="Size">
+      <MainSection.Card
+        cardSize="md"
+        title="Large"
+        description={`Use \`lg\` as the default size within Pinterest products.`}
```

Comment:
We should probably avoid the word "default" here since `'lg'` isn't actually the default value for the prop. Something like "`'lg'` is the recommended size for all Pinterest products"

Reply:
I think we should loop back to this as the recommendation is different between Pinner and M10n. Mind if I backlog this as something for us to revisit as an improvement?

Reply:
And yeah, I think the existing language is deceptive as well, but my thinking is to just keep it as-is and have Ashley/Annie re-evaluate in a future sprint.

Reply:
Right, it's more about not using the word "default" since that has specific meaning within React, therefore leading to unexpected behavior and UI bugs. Anything that avoids using that word is fine by me.

Reply:
Yeah, fair enough. I kept the original sentence and switched out `default` for `standard` - just to keep the changes minimal. Thanks Ryan!